docs/deployment/add-ons/load-balancer/README.md

```diff
@@ -0,0 +1,111 @@
+# Exposing Kubeflow over Load Balancer
+
+This tutorial shows how to expose Kubeflow over a load balancer on AWS.
+
+## Background
+
+Kubeflow does not offer a generic solution for connecting to Kubeflow over a load balancer because this process is highly dependent on your environment/cloud provider. On AWS, we use the [AWS Load Balancer controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/) which satisfies the Kubernetes [Ingress resource](https://kubernetes.io/docs/concepts/services-networking/ingress/) to create an [Application Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html) (ALB). When you create a Kubernetes `Ingress`, an ALB is provisioned that load balances application traffic.
+
+In order to connect to Kubeflow using a LoadBalancer, we need to setup HTTPS. The reason is that many of the Kubeflow web apps (e.g., Tensorboard Web App, Jupyter Web App, Katib UI) use [Secure Cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#restrict_access_to_cookies), so accessing Kubeflow with HTTP over a non-localhost domain does not work.
```

Comment:
What about other components like pipeline, serving ? Do they also not support HTTP ?

Reply:
looks like kserve webapp too based on: https://www.kubeflow.org/docs/external-add-ons/kserve/webapp/#port-forwarding